modules/35_transport/edge_esm/presolve.gms

```diff
@@ -13,7 +13,8 @@ if( (ord(iteration) le 25 and ord(iteration) ge cm_startIter_EDGET and (mod(ord(
     Execute "Rscript -e 'edgeTransport::iterativeEdgeTransport()'";
 
     Execute_Loadpoint 'p35_esCapCost' p35_esCapCost;
-    pm_esCapCost(t,regi,teEs_dyn35)$(t.val gt 2010 AND t.val ge cm_startyear AND t.val le 2100) = p35_esCapCost(t,regi,"%cm_GDPscen%","%cm_demScen%","%cm_EDGEtr_scen%",teEs_dyn35);
```

Comment:
@johannah-pik could you please check, if this is correct? Thanks

Reply:
Why `t.val ge cm_startyear`? Isn't that a bit tautological?